genie-common/src/main/java/com/netflix/genie/common/jobs/JobConstants.java

```diff
@@ -15,7 +15,7 @@
  *     limitations under the License.
  *
  */
-package com.netflix.genie.web.jobs;
+package com.netflix.genie.common.jobs;
```

Comment:
So something like `common-internal` that `web` and `agent` depend on, but not `client`.
I'm in favor of this, but it seems to be beyond the scope of the agent, it's something that affects more the client than anything else, no?

On the other side, if i ever need to move anything major into common that we don't want users to see, i'll create it.
(these are just constant strings, there's not much harm in having them visible no? I agree it would be better not to)

Reply:
Yes something like that. Metacat had something like `server-common` and since we're so great at naming things I'm sure we could come up with something haha. 

We could reintroduce a `genie-core` like we had before.